wormhole-connect/src/store/transferInput.ts

```diff
@@ -49,6 +49,13 @@ export const formatBalance = (
   });
 };
 
+export const formatStringAmount = (amountStr = '0'): string => {
+  const amountNum = parseFloat(amountStr);
+  return amountNum.toLocaleString('en', {
```

Comment:
You can also use [toFixed](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed) instead of locale formatting.

Reply:
Actually, it's not the same, you are looking for maximum decimals, not fixed.